docs/FOLDERS.md

```diff
@@ -24,37 +24,47 @@
 Contains:
  1. Actions, scripts that interact with the services in [services](#services) folder.
  2. Endpoints, exposing the service to other services.
+ 3. Handles the validations and assertions of the requests
 
 ### api/common
 This folder contains all the logic that is shared across multiple files in the parent folder ([api](#api)).
 
 ## config
 This is where configurations are loaded, configurations could be loaded from the environment variables either in production or in development based on where the service has been spawned.
```

Comment:
Configurations can loaded in any environment. But you should basically emphasize two things
1. how configs are grouped into components
2. and the fact that they basically mainly contains configs for our various process types

Reply:
added